File: calc.py
result = ""
myFile = open("calculatorpy.txt", "r")
for line in myFile:
	try:
			if (line.isupper() or line.islower()):
				result += line.strip() + "invalid\n"
			else:
    				result += line.strip() + "=" + str(eval(line)) + "\n"
    	except:
    		result += line.strip() + "invalid\n"
myFile = open("calculatorpy.txt", "w")
myFile.write(result)

File: calc.rb
@result = ""
File.open("calculator.txt", "r").each do | line |
	 	begin
	 	 	@result += line.chomp + "=" + eval(line).to_s + "\n"
	 	 rescue Exception
	 	 	@result += line.chomp + " invalid"
	 	 end 
end
File.open("calculator.txt", "w").puts @result